mdn/developer-portal
# 972 Add streamfield to events page to support custom non-Event content

This changeset addresses the need to link to the sponsorship page _from_ the /events/ page by adding support for free-form rich content (as per our custom StreamField's config)

(Related issue #972)

## Key changes:

## How to test

- Edit the Events page - add some rich content for the Body element and publish or preview the page. Depending on what you add, you'll see something like this:

![Screenshot 2019-12-18 at 13 50 20](https://user-images.githubusercontent.com/101457/71094247-8ea7c680-21a2-11ea-8b65-5b218905b258.png)
